src/core/sys/linux/io_uring.d

```diff
@@ -0,0 +1,414 @@
+/**
+ * D header file for the io_uring interface.
+ * Available since Linux 5.1
```

Comment:
Newline to separate the title from the description. Also some of those symbols are only available since 5.5.